### DISCUSSION TITLE
Make seamless Editing a User preference and not a platform decision

### DISCUSSION THREAD

**Discussion by UKFr-DIZ** [outline/outline#5347] [Ideas]
Hi, 

do you think it'd possible to make seamless editing a user preference ? 
Because we have in our a team a mix of preference how they like editing document and some prefer to read a lot instead of contributing, and they accidentally edit the document.

**Comment by mikew** [comment]
Aye, I was annoyed by [a bug that was apparently missed when this option is disabled](https://github.com/outline/outline/issues/3089#issuecomment-1666641910). Seems like a personal preference and not an org-wide one.

Also, I don't really understand what Seamless Editing is. It seems like it _is_ on for our org, as there is no separate reading / edit modes. We go to any document, click anywhere, and are editing.

**Reply by tommoor** [reply]
I believe I was mistaken and in the end it did not relate to edit mode.
